public/config.template

```diff
@@ -1,21 +1,6 @@
 // DevOps context
 var configUrlApi = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API
-var configUrlApiBeta = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API_BETA
 var configGoogleTagManagerID = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_GOOGLE_TAG_MANAGER_ID
 var configEFOURL = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_EFO_URL
-var configPrimaryColor = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PRIMARY_COLOR
-var configHelpdeskEmail = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HELPDESK_EMAIL
-var configIsPartnerPreview = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_IS_PARTNER_PREVIEW
-var configHideDiseaseSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_DISEASE_SECTION_IDS
-var configPartnerDiseaseSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DISEASE_SECTION_IDS
-var configHideTargetSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_TARGET_SECTION_IDS
-var configPartnerTargetSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_TARGET_SECTION_IDS
-var configHideDrugSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_DRUG_SECTION_IDS
-var configPartnerDrugSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DRUG_SECTION_IDS
-var configHideEvidenceSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_EVIDENCE_SECTION_IDS
-var configPartnerEvidenceSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_EVIDENCE_SECTION_IDS
-var configPartnerColorRange = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_COLOR_RANGE
-var configHideDataTypes = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_DATA_TYPES
-var configPartnerDataTypes = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DATA_TYPES
-var configPartnerDataSources = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DATA_SOURCES
-var configDownloadsURL = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_DOWNLOADS_URL
```

Comment:
Hello @fedde-s, thanks for your PR, it looks good. Only, these variables are needed for our build/release

I'll need to do a quick look at it with the backend team in order to add the variable to our release process.

Reply:
Are you sure? As I explained in the commit message, the code very much suggests that none of these variables do anything anymore as of the change I linked, and the build process already uses the replacement for them.

Reply:
Ah, I do see an oversight in my commit: the broken (and now cleaned-up) variables are still listed in `docs/CUSTOM_CONFIG.md`.

Reply:
For completeness: the patch that removed the logic of reading these variables is commit <https://github.com/opentargets/platform-app/commit/2f40f839e94faffefc12aa5f35a2b71b86d46fb7> in pull request <https://github.com/opentargets/platform-app/pull/487>

Reply:
For completeness, the changes that remove the logic to read the variables (and adds the new way to set it, already used in the Terraform config, see my commit message) are part of pull request <https://github.com/opentargets/platform-app/pull/487>.

Reply:
Thanks, @fedde-s. I review it with the team, and your right.